### DISCUSSION TITLE
pre-commit hook skips even though a renovate.json file exists

### DISCUSSION THREAD

**Discussion by lmilbaum** [renovatebot/pre-commit-hooks#2984] [Q&A] [answered]
`renovate-config-validator............................(no files to check)Skipped`

**Comment by maxbrunet** [comment, marked as answer]
`pre-commit` only runs on the changed files, so if `renovate.json` has no staged changes, it will not be checked.

The CI tests the hook with `--all-files` and it passes:

https://github.com/renovatebot/pre-commit-hooks/runs/8180885985?check_suite_focus=true#step:5:26

Please see the official documentation for more information: https://pre-commit.com/#pre-commit-run
